x-pack/plugins/security_solution/public/hosts/components/host_risk_information/index.tsx

```diff
@@ -81,20 +74,52 @@ export const HostRiskInformation = () => {
         aria-label={i18n.INFORMATION_ARIA_LABEL}
         onClick={handleOnOpen}
         className={HOST_RISK_INFO_BUTTON_CLASS}
-        data-test-subj="open-risk-information-flyout"
+        data-test-subj="open-risk-information-flyout-trigger"
       />
       {isFlyoutVisible && <HostRiskInformationFlyout handleOnClose={handleOnClose} />}
     </>
   );
 };
 
+export const HostRiskInformationButtonEmpty = () => {
```

Comment:
Please wrap `HostRiskInformationButtonEmpty` and `HostRiskInformationFlyout` with React.memo